packages/metal-quartz-icon/src/MetalQuartzIcon.soy

```diff
@@ -1,17 +1,57 @@
 {namespace MetalQuartzIcon}
 
 /**
- * This renders an icon element based on the lexicon icon definition
- * found at http://liferay.github.io/lexicon/content/icons-lexicon/
+ * This renders an icon element based on the Quartz Icon, Quartz Font Awesome
+ * Icon, and Glyphicon definition.
+ * http://getbootstrap.com/components/#glyphicons
  *
  * @param? elementClasses
  * @param spritemap
  * @param symbol
  */
 {template .render}
 	{if isNonnull($symbol) and isNonnull($spritemap)}
-		<svg aria-hidden="true" class="lexicon-icon lexicon-icon-{$symbol}{$elementClasses ? ' ' + $elementClasses : ''}">
-			<use xlink:href="{$spritemap}#{$symbol}" />
-		</svg>
+		{let $quartzIconElementClasses kind="text"}
+			{$elementClasses ? ' ' + $elementClasses : ''}
+		{/let}
+
+		{let $quartzIconVariant kind="text"}
+			{$spritemap == 'quartz-fa' ? 'quartzFa' : $spritemap}
+		{/let}
```

Comment:
@jbalsas, yes they would pass in the classes via `elementClasses` and span will be generated when there is no `spritemap` or `symbol`.